apps/src/templates/progress/SummaryProgressRow.jsx

```diff
@@ -16,127 +16,125 @@ import {
   lessonIsVisible
 } from './progressHelpers';
 
-class SummaryProgressRow extends React.Component {
-  static propTypes = {
-    dark: PropTypes.bool.isRequired,
-    lesson: lessonType.isRequired,
-    levels: PropTypes.arrayOf(levelWithProgressType).isRequired,
+function SummaryProgressRow({
+  dark,
+  lesson,
+  levels,
+  lessonIsHiddenForStudents,
+  lessonIsLockedForUser,
+  lessonIsLockedForAllStudents,
+  viewAs
+}) {
+  // The parent component filters out hidden SummaryProgressRows from the student view,
+  // this check is just to ensure it won't be rendered if it should be hidden for students
+  if (lessonIsHiddenForStudents && viewAs === ViewType.Participant) {
+    return null;
+  }
```

Comment:
Should we take advantage of this refactoring and use the "participant" terminology here instead (in the comment and in the prop name)?

Reply:
That did cross my mind as I was writing.. "student" just seemed so much more clear to me.. and right next to it is `lessonIsHiddenForStudents`. I'm going to leave it for now since I don't want to wait for another drone run, in the future I'll use participant